Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserModel.cpp

```diff
@@ -221,16 +222,98 @@ namespace AzToolsFramework
 
             if (index.isValid())
             {
-                // allow retrieval of mimedata of sources or products only (i.e. cant drag folders or root)
+                // We can only drop items onto folders so set flags accordingly
                 AssetBrowserEntry* item = static_cast<AssetBrowserEntry*>(index.internalPointer());
                 if (item && (item->RTTI_IsTypeOf(ProductAssetBrowserEntry::RTTI_Type()) || item->RTTI_IsTypeOf(SourceAssetBrowserEntry::RTTI_Type())))
                 {
                     return Qt::ItemIsDragEnabled | defaultFlags;
                 }
+                if (item && (item->RTTI_IsTypeOf(FolderAssetBrowserEntry::RTTI_Type())))
+                {
+                    return Qt::ItemIsDragEnabled | Qt::ItemIsDropEnabled | defaultFlags;
+                }
```

Comment:
Nit: Could consolidate this so don't have to check `item` multiple times.

Reply:
Good spot there! Changed.